docs/how-to/make-a-standard-detector.rst

```diff
@@ -0,0 +1,44 @@
+.. note::
+
+   Ophyd async is included on a provisional basis until the v1.0 release and 
+   may change API on minor release numbers before then
+
+Make a StandardDetector
+=======================
+
+`StandardDetector` is an abstract class to assist in creating devices to control EPICS AreaDetector implementations.
```

Comment:
Technically, it doesn't need to be an EPICS areaDetector - since we made PandA also as a `StandardDetector`, no?